### PR TITLE
How to contact core and deal with abandoned feedstocks.

### PR DESCRIPTION
**Pull request by BastianZim**
PR Checklist:

- [x] make all edits to the docs in the `src` directory, not in `docs` or in the html files
- [x] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] put any other relevant information below

This is a follow-up to https://github.com/conda-forge/conda-forge.github.io/pull/1544 as I think it's easier to just add the option to the docs instead of trying to skip it in a comment. Once this is merged, I'll add the link to the bot.

### CODE REVIEW

File: src/maintainer/maintainer_faq.rst [hunk 1]
```diff
@@ -164,3 +164,28 @@ FAQ
...
+.. _mfaq_contact_core:
+
+:ref:`(Q) <mfaq_contact_core>` **How can I contact conda-forge/core?**
+
+  When in an issue or PR, you can contact `conda-forge/core <https://conda-forge.org/docs/orga/governance.html#teams-roles>`_ by simply mentioning ``@conda-forge/core`` in a comment.
+  If you don't receive an an answer after a couple of days, feel free to reach out to us via the public `gitter <https://gitter.im/conda-forge/conda-forge.github.io>`_ channel.
+
+  .. note::
+
+    Due to a GitHub limitation, this is disabled for new members.
```
Comment (jaimergp) [inline src/maintainer/maintainer_faq.rst:178]: I recall the bot had a [command](https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-ping-team) to ping teams on behalf of new users. Is that still around? Does it support pinging core directly?

Comment (BastianZim) [reply src/maintainer/maintainer_faq.rst:178]: It just pings the team of the feedstock which is basically the same as core on `staged-recipes` but not on other repos.

Comment (jaimergp) [reply src/maintainer/maintainer_faq.rst:178]: Got it. I wonder if the rest of Core thinks it's a good idea to add a new command for this use case?

Comment (isuruf) [reply src/maintainer/maintainer_faq.rst:178]: you can already do @conda-forge-admin, please ping conda-forge/core

Comment (BastianZim) [reply src/maintainer/maintainer_faq.rst:178]: Ahh ok then that should probably be added to https://conda-forge.org/docs/maintainer/infrastructure.html#admin-web-services - I'll update it.

**Comment by isuruf** [comment]
@conda-forge-admin, please ping conda-forge/core

File: src/maintainer/maintainer_faq.rst (hunk 1 above)
Comment (BastianZim) [reply src/maintainer/maintainer_faq.rst:178]: I cannot find anything in https://github.com/conda-forge/conda-forge-webservices/blob/main/conda_forge_webservices/commands.py - are you sure?

**Comment by BastianZim** [comment]
One sec

**Comment by BastianZim** [comment]
Have tried it on https://github.com/conda-forge/duet-python-feedstock/issues/1 but haven't gotten an answer yet.

**Comment by BastianZim** [comment]
I think it doesn't work @isuruf

**Comment by BastianZim** [comment]
@isuruf Anything else that should be added/changed?

**Pull request merged**